src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,24 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <>
+    <div
+      className="PostInfo"
+    >
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{post.title}</h3>
+
+        <p>
+          {' Posted by  '}
+          <UserInfo user={post.user} />
```

Comment:
```suggestion
          user && <UserInfo user={user} />
```
Consider implementing a check if a user exists